### DISCUSSION TITLE
Components

### DISCUSSION THREAD

**Discussion by mvrk33** [mvrk33/AgroBot#10] [Announcements]
- [x]  [RPi 4 1GB](https://robu.in/product/raspberry-pi-4-model-b-with-1-gb-ram/)
- [x] [64GB Memory card](https://robu.in/product/sandisk-micro-sdxc-ush-i-64gb-class-10-memory-card/)
- [x]  [ Card reader](https://robu.in/product/3-in-1-usb-type-c-otg-card-reader/)
-  [x] [ BMX160 9-axis Sensor](https://robu.in/product/dfrobot-fermion-bmx160-9-axis-sensor-breakout/)
-  [x] [HW-290/GY-87 10DOF MPU6050+HMC5883L+BMP180](https://robu.in/product/mpu6050hmc5883lbmp180-10dof-3-axis-gyro-3-axis-acceleration-3-axis-magnetic-field-air-pres/)
-  [x] FlySky CT6B 2.4GHz 6CH Transmitter with FS-R6B Receiver
- [x] Raspberry Pi Camera Module 3
- [x] Raspberry Pi Camera Module 3 Wide
- [x] M3*20MM Male to Female Nylon Hex Spacer-10Pcs.
- [x] M3*15MM Female to Female Nylon Hex Spacer-25 Pcs.
- [x] M3*20MM Female to Female Nylon Hex Spacer-25 Pcs.

**Comment by krishnavamsi333** [comment]
- [ ] [gold plated type connectors](https://robu.in/product/hxt-4mm-gold-connector-w-protector-1pcsset/)
- [ ] [Buck Converter ](https://robu.in/product/ultra-small-size-dc-dc-5v-3a-bec-power-supply-buck-step-down-module/)
- [ ] [adjustable buck converter ](https://robu.in/product/xl4005-dc-dc-5-32v)
- [ ] [Tyres for 6 mm Shaft Geared Dc Motor](https://robocraze.com/products/10-x-2-cm-gear-motor-robot-wheel-tyres-for-6-mm-shaft-geared-dc-motor-4-pieces)
- [ ] [M3 X 5 mm Bolt- 25 Pcs.](https://robu.in/product/easymech-ss-304-pan-philips-head-m3-x-5-mm-bolt-25-pcs/)

- [ ] [M3 NUTS 304 ](https://robokits.co.in/robot-parts/nut-bolts-standoffs/nuts/m3-nuts-304-stainless-steel-moq-50-pcs?cPath=67_60_460&)
- [ ] [M3 FLAT WASHER](https://robokits.co.in/robot-parts/nut-bolts-standoffs/washers/m3-flat-washer-304-stainless-steel-moq-50-pcs)
- [ ] [Sheet Plate 300mm *200mm * 0.5mm]

**Comment by snehakalakonda03** [comment]
[servo motor](https://robu.in/product/towerpro-sg90-9gm-1-2kg-180-degree-rotation-servo-motor-good-quality/)
[ultrasonic senssor](https://robu.in/product/hc-sr04-ultrasonic-range-finder-cartoon-ultrasonic-sensor-mounting-bracket/)

**Comment by poojithanagasarapu** [comment]
[Wheel encoder](https://robu.in/product/speed-measuring-sensor-groove-coupler-module-arduino/)

**Comment by gurukavya123** [comment]
[Voltage Detection Sensor Module 25V](https://robu.in/product/voltage-sensor-module/)

**Comment by mvrk33** [comment]
> * [x]   [RPi 4 1GB](https://robu.in/product/raspberry-pi-4-model-b-with-1-gb-ram/)
> * [x]  [64GB Memory card](https://robu.in/product/sandisk-micro-sdxc-ush-i-64gb-class-10-memory-card/)
> * [x]   [ Card reader](https://robu.in/product/3-in-1-usb-type-c-otg-card-reader/)
> * [x]  [ BMX160 9-axis Sensor](https://robu.in/product/dfrobot-fermion-bmx160-9-axis-sensor-breakout/)
> * [x]  [HW-290/GY-87 10DOF MPU6050+HMC5883L+BMP180](https://robu.in/product/mpu6050hmc5883lbmp180-10dof-3-axis-gyro-3-axis-acceleration-3-axis-magnetic-field-air-pres/)
> * [x]  FlySky CT6B 2.4GHz 6CH Transmitter with FS-R6B Receiver
> * [x]  Raspberry Pi Camera Module 3
> * [x]  Raspberry Pi Camera Module 3 Wide
> * [x]  M3*20MM Male to Female Nylon Hex Spacer-10Pcs.
> * [x]  M3*15MM Female to Female Nylon Hex Spacer-25 Pcs.
> * [x]  M3*20MM Female to Female Nylon Hex Spacer-25 Pcs.

delivered

**Reply by mvrk33** [reply]
Qty. | Products | Total
-- | -- | --
1 | Samsung Cell Premium Li-Ion 11.1V 5200mAh With Charge ProtectionCharger - RKI-1369 IMAX B6-AC CHARGER/DISCHARGER 1-6 CELLS | ₹4,419.00
3 | HC-SR04 Ultrasonic Sensor with Mounting Bracket | ₹315.00
4 | Rhino GB37 12V 60RPM 10.4Kgcm DC Geared Encoder Servo MotorMounting Bracket - RKI-1353 HIGH QUALITY BRACKET FOR HIGH TORQUE/SIDE SHAFT MOTORMotor Driver - RMCS-2303 UART ASCII Encoder DC Motor Driver | ₹6,188.00
5 | Six in One Multi Function Module CP2102 USB to TTL 485 232 Conversion 3.3V/5V Compatible | ₹910.00
50 | M3 Nuts 304 Stainless Steel MOQ 50 pcs | ₹186.00
50 | M3 Flat Washer 304 Stainless Steel MOQ 50 Pcs | ₹120.00
15 | M3 x 30 mm Socket Head Cap Stainless Steel 304 Bolt MOQ 15 Pcs | ₹195.00

**Comment by krishnavamsi333** [comment]
> - [ ] [gold plated type connectors](https://robu.in/product/hxt-4mm-gold-connector-w-protector-1pcsset/)
> - [ ] [Buck Converter ](https://robu.in/product/ultra-small-size-dc-dc-5v-3a-bec-power-supply-buck-step-down-module/)
> - [ ] [adjustable buck converter ](https://robu.in/product/xl4005-dc-dc-5-32v)
> - [ ] [Tyres for 6 mm Shaft Geared Dc Motor](https://robocraze.com/products/10-x-2-cm-gear-motor-robot-wheel-tyres-for-6-mm-shaft-geared-dc-motor-4-pieces)
> - [ ] [M3 X 5 mm Bolt- 25 Pcs.](https://robu.in/product/easymech-ss-304-pan-philips-head-m3-x-5-mm-bolt-25-pcs/)
> 
> - [ ] [M3 NUTS 304 ](https://robokits.co.in/robot-parts/nut-bolts-standoffs/nuts/m3-nuts-304-stainless-steel-moq-50-pcs?cPath=67_60_460&)
> - [ ] [M3 FLAT WASHER](https://robokits.co.in/robot-parts/nut-bolts-standoffs/washers/m3-flat-washer-304-stainless-steel-moq-50-pcs)
> - [ ] [Sheet Plate 300mm *200mm * 0.5mm]
> 

Pending components
